CONTRIBUTING.md

```diff
@@ -216,6 +217,24 @@ Read cargo's documentation for more details: https://doc.rust-lang.org/cargo/ref
   open $"($nu.temp-path)/nu-($nu.pid).log"
   ```
 
+### Experimental options
+
+We sometimes add new behavior behind [**experimental options**](https://www.nushell.sh/blog/2025-07-23-nushell_0_106_0.html#experimental-options-toc). 
+These are opt-in (or opt-out) flags that let us try out changes without affecting everyone by default.
+
+If you introduce a new experimental option, you must also:
+
+1. **Open a tracking issue** using the [Experimental Option Tracking template](https://github.com/nushell/nushell/issues/new/choose).  
```

Comment:
maybe we should make sure the PR is actually something we want so we don't get drive-by experimental option tracking issues for things that are never merged